filter/etc/e-smith/templates/etc/httpd/admin-conf.d/rspamd.conf/10base

```diff
@@ -1,25 +1,34 @@
 {
     use esmith::ConfigDB;
-    use NethServer::TrustedNetworks;
     use esmith::util;
     my $confdb = esmith::ConfigDB->open;
     my $rspamd = $confdb->get('rspamd') or die "No rspamd db entry found\n";
     my $alias = $rspamd->prop('alias') || "";
+    my $user = join (' ', split (',',$admins{'user'}));
 
     # initialize alias if needed
     if ($alias eq "") {
         $alias = esmith::util::genRandomHash();
         $confdb->set_prop('rspamd','alias',$alias);
     }
 
-    my $trustedNetworks = join(" ", NethServer::TrustedNetworks::list_cidr());
-
 $OUT .= qq(
+<IfModule !authnz_pam_module>
+    LoadModule authnz_pam_module modules/mod_authnz_pam.so
+</IfModule>
+
 <Location /$alias>
+    ProxyAddHeaders off
     ProxyPass  http://localhost:11334
     ProxyPassReverse http://localhost:11334
     ProxyPreserveHost On
-    Require ip $trustedNetworks
+    AuthName "rspamd"
+    AuthType Basic
+    AuthBasicProvider PAM
+    AuthPAMService rspamd
+    <RequireAll>
```

Comment:
I'd remove the requireall block. We should grant access to either admin user or members of the admin group. To accomplish this a possible approach could be inserting here a `Require valid-user` and restrict the pam module configuration with `pam_succeed_if` or similar